collection-scripts/gather_audit_logs

```diff
@@ -36,15 +40,15 @@ paths=(openshift-apiserver kube-apiserver oauth-apiserver etcd oauth-server)
 for path in "${paths[@]}" ; do
   output_dir="${BASE_COLLECTION_PATH}/audit_logs/$path"
   mkdir -p "$output_dir"
-  oc adm node-logs --role=master --path="$path" | \
+  oc adm node-logs ${node_log_collection_args:+"$node_log_collection_args"} --role=master --path="$path" | \
   tee "${BASE_COLLECTION_PATH}/audit_logs/$path.audit_logs_listing" | \
   grep -v ".terminating" | \
   grep -v ".lock" | \
   sed "s|^|$path $output_dir |"
 done | \
 xargs --max-args=4 --max-procs=45 bash -c \
   'echo "INFO: Started  downloading $1/$4 from $3";
-  oc adm node-logs $3 --path=$1/$4 | gzip > $2/$3-$4.gz;
+  oc adm node-logs ${node_log_collection_args} $3 --path=$1/$4 | gzip > $2/$3-$4.gz;
```

Comment:
this doesn't actually work - `xargs` does not have access to env vars, so `node_log_collection_args` will always be empty here. right now it doesn't make a difference because the `--since` flag only affects jounal logs for `oc adm node-logs`. 
I'm fixing it anyway so we're future proof.

Reply:
last push fixes this - @ardaguclu can you take another look? This is the last one, I promise 😅

Reply:
No worries :). Tagged.